XenModel/Messages.resx

```diff
@@ -11627,6 +11627,9 @@ Check your settings and try again.</value>
   <data name="UPDATES_WIZARD_CANNOT_MIGRATE_VM_LICENSE_REASON" xml:space="preserve">
     <value>Cannot migrate VM '{0}' due to license restrictions.</value>
   </data>
+  <data name="UPDATES_WIZARD_CANNOT_MIGRATE_VM_SUSPEND_REASON" xml:space="preserve">
+    <value>Cannot migrate VM '{0}', because it lacks the ability to suspend. Please ensure it has up-to-date drivers installed.</value>

```

Comment:
Also, the second part of the message ("Please ensure it has up-to-date drivers installed") suggests that installing the drivers will fix the problem, but at the same time we offer "Shut down" as the solution, which is confusing.